### PR TITLE
refactor(highlight): convert code block declarations from tags to Markdown syntax extensions

### PR DESCRIPTION
**Pull request by normal-coder**
## What does this Pull Request do

- Convert code block declarations in Pipeline documents from tags to Markdown syntax extensions
- not changed for other specific content

## Why do this

- In Drone's documentation, there are numerous references to code block highlighting declarations similar to `{{<highlight yaml "linenos=table" >}} ... {{</ highlight >}}`, which is Bright rendering of Markdown documents has a huge experience barrier, and at the same time makes the reading experience of the document repository code extremely poor.
- In the document, there is another way of writing code block highlighting `` ```yaml {linenos=table} ... ``` ``, the function is to keep the code block style consistent with the above highlighted declaration, This writing method conforms to the Markdown syntax style and is more concise. It is recommended to use this method. This is also the core purpose of this Pull Request.
- In fact, the entire document should have a unified code style for easy maintenance, this Pull Request only deals with the documentation of the Pipeline module

### CODE REVIEW

**Comment by ShruthiHubb** [comment]
Hi @normal-coder we will have this reviewed every Tuesday and team will get back to you after learning the feasibility of request. Feel free to reach us out on [community slack](https://harnesscommunity.slack.com/ssb/redirect#/shared-invite/email) for quick help.

Thanks

**Comment by normal-coder** [comment]
> Hi @normal-coder we will have this reviewed every Tuesday and team will get back to you after learning the feasibility of request. Feel free to reach us out on community slack for quick help.
> 
> Thanks

It looks great. Due to the many changes to the current position, I am patient enough to wait for the communication with you, and I will join the Slack community later.

**Comment by jimsheldon** [comment]
Thanks for joining Drone Office Hours today to discuss this.

As I mentioned, please give us a new pull request with a change to a single file, then we can see how it looks and go from there.

Thanks again!

**Comment by normal-coder** [comment]
@ShruthiHubb @jimsheldon , I've adjusted the commit to a small number of file changes so that it's easier to see the impact of their changes. 

For other changed files, I will submit a separate Pull Request for adjustment after the Pull Request is merged. Looking forward to the follow-up communication and feedback.

File: content/pipeline/configuration.md [hunk 1]
```diff
@@ -124,18 +124,19 @@ type: docker
...
 steps:
-- name: build
+
+* name: build
   image: node
   commands:
-  - npm install
-  - npm test
-{{< / highlight >}}
+  * npm install
```
Comment (jimsheldon) [inline content/pipeline/configuration.md:131]: @normal-coder this line, as well as 128 and 132 should start with `-`, `*` is invalid yaml

**Review by jimsheldon — CHANGES_REQUESTED** [review]
Please make these changes

**Comment by normal-coder** [comment]
It seems that the Pull Request has not been reviewed for a long time. Any questions? @jimsheldon

**Comment by ShruthiHubb** [comment]
@normal-coder let me check for the updates.

Thanks!

**Review by d1wilko — APPROVED** [review]
LGTM

**Pull request merged**